### PR TITLE
Add generator & move handlebars files into framework folder

### PR DESCRIPTION
**Pull request by swcolegrove**

### CODE REVIEW

File: src/frameworks/handlebars/index.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    webpack: '^4.41.2',
+    'webpack-cli': '^3.3.10',
+    'webpack-dev-server': '^3.11.0',
+  },
+};
+
+/* eslint-enable */
+
+module.exports = async function handlebars() {
+  this.log('Coyping templates');
```
Comment (swcolegrove) [inline src/frameworks/handlebars/index.js:82]: im gonna change this so it creates a new directory for the project similar to other framework cli tools

**Review by davidragsdale — CHANGES_REQUESTED** [review]
Just a few changes

File: src/frameworks/handlebars/index.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+/* eslint-disable no-useless-escape */
+const packageJsonScripts = {
+  scripts: {
+    build: 'npm-run-all build:clean build:html build:js',
+    'build:clean': 'rm -rf ./dist/ && rm -rf ./public/',
+    'build:html': 'env-cmd handlebars-transpile',
+    'build:js': 'webpack --config webpack.prod.js --mode production',
+    dev: 'npm-run-all -p watch:*',
+    'watch:html': 'nodemon -e hbs --exec \"npm run build:html\"',
```
[the remark this answers is not in the document]
Comment (swcolegrove) [reply src/frameworks/handlebars/index.js:13]: Yep i tested this, if we try to escape the escapes we end up with both escapes still in the destination package.json

File: src/frameworks/handlebars/index.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  await fse.mkdir(this.destinationPath(projectName));
+  process.chdir(projectName); // change directory to new folder
+
+  this.log('Coyping templates');
+  await this.copy(
+    path.join(__dirname, './template'),
+    this.destinationPath(),
+  );
+
+  this.log('Coyping common files');
```
[the remark this answers is not in the document]
Comment (swcolegrove) [reply src/frameworks/handlebars/index.js:98]: me good at spell

File: src/util/destinationPath.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const path = require('path');
+
+module.exports = function destinationPath(folder = '') {
```
[the remark this answers is not in the document]
Comment (swcolegrove) [reply src/util/destinationPath.js:3]: I think leaving it as empty string is fine, it does not always need a folder/file name if you are copying directly into the root folder for instance

**Review by davidragsdale — APPROVED** [review]
LGTM. 🚀

**Review by tgardnerFusion — APPROVED** [review]
Looks gouda 🧀

**Pull request merged**